mysql-test/suite/rocksdb/t/bulk_load_errors.test

```diff
@@ -21,21 +27,32 @@ INSERT INTO t1 VALUES(2);
 INSERT INTO t1 VALUES(20);
 INSERT INTO t1 VALUES(21);
 
-# This last crashes the server (intentionally) because we can't return any
-# error information from a SET <variable>=<value>
---exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
---error 2013
+--error ER_OVERLAPPING_KEYS
 SET rocksdb_bulk_load=0;
 
---exec grep "RocksDB: Error 197 finalizing last SST file while setting bulk loading variable" $MYSQLTEST_VARDIR/log/mysqld.1.err | cut -d] -f2
---exec echo "" >$MYSQLTEST_VARDIR/log/mysqld.1.err
+SHOW VARIABLES LIKE 'rocksdb_bulk_load';
+
+SELECT * FROM t1;
+
+--let SEARCH_PATTERN=RocksDB: Error 198 finalizing last SST file while setting bulk loading variable
+--source include/search_pattern_in_file.inc
+--exec echo " " > $SEARCH_FILE
```

Comment:
Restart again with a different SEARCH_FILE instead?

Reply:
Slow the test down with no gain?

Reply:
In order not to limit portability of the testcase: IIRC, on Windows the "exec echo" would fail to truncate the file because mysqld opens it without allowing shared I/O for any subsequent process trying to open it again. But, this change would be more of a favor for MariaDB MyRocks, as they are the ones with Windows build, and maybe they are in a better position to suggest a patch here.

Reply:
Hmm, true, that is a fair point.

Reply:
OK, changed to restarting the server with fresh logs for each search, test takes ~75x longer to run (was ~80ms locally before but is now ~6100ms) but should be safe for any 'doze users.

Reply:
The first and the final restarts are required for error log handling correctness on all platforms, thus I hope that Windows adds only a ~2x slowdown :)